.rubocop.yml

```diff
@@ -111,10 +107,10 @@ RSpec/MessageSpies:
   EnforcedStyle: receive
 
 RSpec/ExampleLength:
-  Max: 16
+  Max: 31
 
 RSpec/NestedGroups:
-  Max: 4
+  Max: 6
```

Comment:
Verify with the project owner before making changes to the rubocop settings for Hyku.